src/python-problem/problem/watch.py

```diff
@@ -15,9 +16,13 @@ class ProblemWatcher(object):
 
     def __init__(self, auth):
         import dbus
-        import gobject
         from dbus.mainloop.glib import DBusGMainLoop
 
+        if 'gi' in sys.modules.keys():
+            from gi.repository import GObject as gobject
+        else:
+            import gobject
+
```

Comment:
Do we really need to this exercise or can we just replace `import gobject` with `from gi.repository import GObject as gobject`?